build.gradle

```diff
@@ -130,18 +141,24 @@ dependencies {
     implementation "org.grails.plugins:ala-bootstrap3:4.4.0" //, noCache
     implementation "org.grails.plugins:ala-admin-plugin:2.3.0" //, noCache
 
-    implementation 'au.org.ala.plugins:openapi:1.1.0'
     // db-migration
     implementation 'org.liquibase:liquibase-core:4.19.0'
     implementation('org.grails.plugins:database-migration:4.2.0') {
         // spring-boot-cli exclusion required since Grails5 upgrade to prevent NullPointerException Error:  https://github.com/grails/grails-database-migration/issues/268
         exclude module: 'spring-boot-cli'
     }
-}
 
-webdriverBinaries {
-    chromedriver "$chromeDriverVersion"
-    geckodriver "$geckodriverVersion"
+
+    implementation 'au.org.ala:ala-cas-client:3.0.0'
+    implementation ("au.org.ala.names:ala-namematching-client:1.8.1") {
+        exclude group: "com.squareup.okhttp3", module: "okhttp"
+        exclude module: "jsr250-api"
+        exclude module: "sisu-guava"
+        exclude module: "cache2k-api"
+    }
+    implementation 'au.org.ala.plugins:openapi:1.1.0'
+
+    implementation 'org.gbif:gbif-common:0.37'
```

Comment:
Not used.

Reply:
implementation 'au.org.ala.plugins:openapi:1.1.0'” is still used and give below exception - unable to resolve class io.swagger.v3.oas.annotations.Parameter.

Removed - 'org.gbif:gbif-common:0.37'